opentelemetry-api/src/opentelemetry/metrics/_internal/__init__.py

```diff
@@ -47,7 +47,10 @@
 from threading import Lock
 from typing import List, Optional, Sequence, Set, Tuple, Union, cast
 
-from opentelemetry._metrics._internal.instrument import (
+from opentelemetry.environment_variables import (
+    _OTEL_PYTHON_METER_PROVIDER as OTEL_PYTHON_METER_PROVIDER,
```

Comment:
`_OTEL_PYTHON_METER_PROVIDER` in `opentelemetry.environment_variables` should get the _ removed too?

Reply:
Fixed